src/igvfd/tests/data/inserts/analysis_set.json

```diff
@@ -57,5 +57,27 @@
         "uuid": "f1149b20-2066-49d8-bbd0-7f77f642cae3",
         "release_timestamp": "2024-08-06T12:34:56Z",
         "file_set_type": "intermediate analysis"
+    },
+    {
+        "aliases": [
+            "igvf:analysis_set_with_workflow"
+        ],
+        "submitter_comment": "Comment for an analysis set.",
+        "description": "An analysis set with work flow info.",
+        "award": "HG012012",
+        "lab": "j-michael-cherry",
+        "documents": [
+            "igvf:Antibody_characterization_standards_IGVF"
+        ],
+        "status": "released",
+        "uuid": "ff3e81bd-b427-492d-808b-e153b067b664",
+        "samples": [
+            "igvf:treated_primary_cells"
+        ],
+        "release_timestamp": "2024-03-06T12:34:56Z",
+        "file_set_type": "intermediate analysis",
+        "workflows": [
```

Comment:
Wouldn't add it to the insert since its a calculated property

Reply:
Done